app/serviceworker/update.js

```diff
@@ -0,0 +1,7 @@
+self.addEventListener('install', (event) => {
+  event.waitUntil(self.skipWaiting());
+});
+
+self.addEventListener('activate', (event) => {
+  event.waitUntil(self.clients.claim());
+});
```

Comment:
I tried something fancier, like displaying a notification on which the user would click before taking over the control, but it was not working as unactivated service workers can't start a notification, and moreover the service worker itself could be put to sleep after the "install" process if we don't use waitUntil.